src/lib/accessibility/Behaviors/Button/buttonBehavior.ts

```diff
@@ -3,14 +3,14 @@ import { Accessibility } from '../../interfaces'
 /**
  * @description
  *  Adds role='button' if element type is other than 'button'. This allows screen readers to handle the component as a button.
- *  Adds attribute 'aria-disabled=true' based on the property 'disabled'.
+ *  Adds attribute 'aria-disabled=true' based on the property 'disabled'. This can be overriden by providing 'aria-disabled' property directly to the component.
  */
 
 const buttonBehavior: Accessibility = (props: any) => ({
   attributes: {
     root: {
       role: props.as === 'button' ? undefined : 'button',
-      'aria-disabled': !!props['disabled'],
+      'aria-disabled': props['aria-disabled'] || props['disabled'],
```

Comment:
The description indicates that passing `aria-disabled` will override the default behavior, however, the logic here only works if you pass `true`.  The short circuit will not respect a value of `false`.

An override would respect any value the user passes:

```suggestion
      'aria-disabled': typeof props['aria-disabled'] !== 'undefined' ? props['aria-disabled'] : props['disabled'],
```


Or possibly only not `undefined` nor `null` values:

```suggestion
      'aria-disabled': !_.isNil(props['aria-disabled']) ? props['aria-disabled'] : props['disabled'],
```

Not sure what the expected intent is here.  My intuition says the second example would seem most correct to a React user, since `null` and `undefined` are both considered "not present" in React.